chapters/ar/_toctree.yml

```diff
@@ -0,0 +1,11 @@
+- title: 0. الإعداد
```

Comment:
We should remove this section as the docs won't render if we don't have the corresponding MDX file

Reply:
Thanks @lewtun I will add only the sections that I worked on

Reply:
Great. I will continue working on other sections of chapter 2 in the same manner.